foundry.toml

```diff
@@ -25,7 +25,7 @@ depth = 10
 [profile.optimized-build]
 via_ir = true
 test = 'src'
-optimizer_runs = 15000
```

Comment:
We may want to revisit codesize in the future. It's unfortunate to reduce optimizer runs, but it's still more than enough.